src/bidiMapper/domains/network/NetworkRequest.ts

```diff
@@ -230,7 +230,14 @@ export class NetworkRequest {
   onRequestPaused(
     params: Protocol.Fetch.RequestPausedEvent,
     networkStorage: NetworkStorage
-  ) {
+  ): void {
+    if (this.#isIgnoredEvent()) {
+      void this.continueRequest(params.requestId).catch(() => {
```

Comment:
No `await`?

Reply:
Here no difference will be observed so explicitly opting in for `void` to make the code easier to understand.